lib/features/email/domain/state/get_opened_email_from_cache_state.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:core/presentation/state/failure.dart';
+import 'package:core/presentation/state/success.dart';
+import 'package:tmail_ui_user/features/email/domain/model/detailed_email.dart';
+
+class GetOpenedEmailLoading extends UIState {}
+
+class GetOpenedEmailSuccess extends UIState {
+  final DetailedEmail detailedEmail;
+
+  GetOpenedEmailSuccess(this.detailedEmail);
+
+  @override
+  List<Object?> get props => [detailedEmail];
+}
+
+class GetOpenedEmailFailure extends FeatureFailure {
+
+  GetOpenedEmailFailure(dynamic exception) : super(exception: exception);
+
+  @override
+  List<Object?> get props => [exception];
+}
```

Comment:
we really need the separate states for `get content interactor`?

Reply:
No need, I remove it